From 6a6091a4d35399ba6135de8e2c0244e418e64187 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Mon, 16 Jan 2023 16:19:50 +0800
Subject: [PATCH] Revert "Linux 5.10.163"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 19ff2d645f7a9626146b6b3ba698488d60aa04de.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 98fc6e7fd41d..704709f34da3 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 163
+SUBLEVEL = 162
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

